requirements.txt

```diff
@@ -1,3 +1,3 @@
-folium==0.2.1
+folium==0.9.1
```

Comment:
I'm inclined to suggest `folium>=0.9.1` (if we later discover that there is some known problem with a future version of folium we can change this to pin a particular version).  We might pin `folium==0.9.1` on datahub but I'm not sure it is necessary or desirable in `requirements.txt`.